planning/behavior_path_planner/include/behavior_path_planner/utils/avoidance/avoidance_module_data.hpp

```diff
@@ -93,7 +93,7 @@ struct AvoidanceParameters
   bool use_opposite_lane{true};
 
   // enable update path when if detected objects on planner data is gone.
```

Comment:
should be updated?

Reply:
I updated breif description and fix param name in doc.